### DISCUSSION TITLE
/opening-folders-visual-studio-code-extension/

### DISCUSSION THREAD

**Discussion by comment-migrator** [estruyf/eliostruyf-comments#599] [Comments]
Opening folders in Visual Studio Code from an extension
        
[https://www.eliostruyf.com/opening-folders-visual-studio-code-extension/](https://www.eliostruyf.com/opening-folders-visual-studio-code-extension/)

**Comment by Askhento** [comment]
The reverse would be something like this:
```js
const yourUri = vscode.Uri.file(yourPath);
vscode.commands.executeCommand("revealFileInOS", yourUri);
```
Referenced your article [at stackoverflow](https://stackoverflow.com/questions/72194573/how-to-open-a-file-externally-using-built-in-commands/78308133#78308133)
